automation/vars/default.yaml

```diff
@@ -0,0 +1,46 @@
+---
+vas:
+  hci:
+    repo: https://github.com/openstack-k8s-operators/architecture.git
+    branch: main
+    stages:
+    - path: examples/va/hci/control-plane/nncp
+      validation: "oc wait nncp --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured --timeout=60" 
+      values:
+      - name: network-values
+        src_file: network_values.yaml
+        dest_file: values.yaml
+
+    - path: examples/va/hci/control-plane
```

Comment:
this one looks like a duplication, only diff being the `validation` I think?

Reply:
We have separated the networking and `OpenStackControlPlane` configuration into two different steps due to errors that can occur when both are run in one `oc apply` batch.  Thus the first stage is for the networking and the second stage for the `OpenStackControlPlane`.  What might be confusing is that they both use the same `values.yaml`, but this is necessary because both need to access the networking data.

Reply:
Indeed, saw it while reading the ctplane deployment doc - no problem then.